OZprivate/rawJS/OZTreeModule/src/render/renderer.js

```diff
@@ -73,6 +76,27 @@ function refresh(root) {
   }
 }
 
+function update_cursor() {
```

Comment:
This could feasibly live in the controller too, happy to go with whatever was more in line with the intention.

Reply:
Haha, yeah I think to start with I'd tried 'zoom-in' cursor for 'tap2zoom' etc. but decided it looked silly so the switch statement is spare now. Have simplified and also added in a little check to make sure we don't steamroll over the mouse interactor's 'zoom-in'/'zoom-out' cursor setting.